skimage/color/tests/test_colorconv.py

```diff
@@ -77,9 +77,14 @@ class TestColorconv():
                           ])
 
     # RGBA to RGB
-    def test_rgba2rgb_conversion(self):
+    @pytest.mark.parametrize("channel_axis", [0, 1, 2, -1])
```

Comment:
In the following tests, may be can we test all the negative axis value range (not only -1)
```suggestion
    @pytest.mark.parametrize("channel_axis", [0, 1, 2, -1, -2, -3])
```